### DISCUSSION TITLE
Why does Py5Vector have a `.copy` property instead of a `.copy()` method?

### DISCUSSION THREAD

**Discussion by villares** [py5coding/py5generator#558] [Q&A] [answered]
Is it for performance?

 I know Py5Vector implementation uses NumPy arrays inside, and an np.array object also has a `.copy()` mehod... as old *PVector*  objects did. 

Just curious...

**Comment by hx2A** [comment, marked as answer]
I did that to simplify the coding when you need to do something with a copy. For example, if `v1 = PyVector(1, 2, 3)`, then you can access `v1.copy.rotate(py5.PI / 2, dim=1)` without rotating `v1`. I guess it wouldn't have been too much trouble if you needed to also include the `()` but this makes the code a bit easier to type and is similar to `norm`, which works the same way.
